app/assets/config/manifest.js

```diff
@@ -1,4 +1,4 @@
 //= link_tree ../images
-//= link_directory ../stylesheets .css
 //= link_tree ../../javascript .js
 //= link_tree ../../../vendor/javascript .js
+//= link_tree ../builds
```

Comment:
- [ ] [REQUIRED] Kindly add 
`//= link_tree ../css`
This will fix some of the errors that are making the page break.